Common/Libraries/DiscosBackendProtocol/include/command.hpp

```diff
@@ -27,6 +27,7 @@ namespace DiscosBackend{
             static Message setFilename(string filename);
             static Message convertData(){return Message(REQUEST, "convert-data");};
             static Message getRms(){return Message(REQUEST, "get-rms");};
+	    static Message setNodding(int feed1, int feed2);
```

Comment:
This is inconsistent with the discos-backend protocol implementation. It should be changed to setEnable as shown [here](https://github.com/discos/discos-backend/blob/master/src/handlers.py#L98).